src/login/pam_systemd.c

```diff
@@ -936,30 +936,21 @@ _public_ PAM_EXTERN int pam_sm_open_session(
         if (r != PAM_SUCCESS)
```

Comment:
I'm thinking if we should allow `TakeDevice` and friends only for non-manager sessions. Otherwise this change would potentially break existing device access models?